pages/api/is-admin.tsx

```diff
@@ -0,0 +1,16 @@
+import { initializeAdmin } from "@/lib/firebase-admin";
+const admin = require("firebase-admin");
+
+export const isAdmin = async (token: string): Promise<boolean> => {
+  await initializeAdmin();
+  if (!token || Object.keys(token).length > 0) {
+    const decodedToken = await admin.auth().verifyIdToken(token);
+    return decodedToken.admin;
```

Comment:
we could probably just use the `verifyAdminToken` function here right? That function basically has the same logic except it also throws an error for token expiration

Reply:
could also add the `!token` logic to the `verifyAdminToken` function, maybe we throw an error if no token is provided

Reply:
Nah you know what, I can get to this. Easy.